lib/dams/controller_helper.rb

```diff
@@ -675,8 +675,12 @@ def audit( id = "unknown" )
     def referrer_controller( request )
       uri = URI(request.referrer || "")
       if uri.host == request.host
```

Comment:
Would it be possible to remove the begin/rescue block, and instead update the conditional check to include something like the following to ensure that the referrer URL is under the /dc/ application path?

`
if( uri.host == request.host && uri.path.start_with? '/dc' )
`